styles.css

```diff
@@ -87,3 +89,257 @@ body {
   padding-left: 10px;
   padding-right: 10px;
 }
+
+.Middle-page {
+  background-color: #e5e5e5;
+  height: 100%;
+  margin-top: -35px;
+  padding-left: 15px;
+  padding-right: 15px;
+  display: grid;
+}
+
+.Middle-page > h1 {
+  text-align: center;
+  padding-top: 100px;
+  padding-bottom: -10px;
+  font-size: 40px;
+  font-family: 'Crete Round', serif;
```

Comment:
Please do not repeat this line of code and use the DRY rule. You can make a class and use it as many times as it is needed.